metadata/index.html

```diff
@@ -921,9 +890,11 @@
               The <a href="#column-name"><code>name</code></a> properties of the column descriptions MUST be unique within a given table description.
             </p>
             <p>
-              When an array of column descriptions B is <a>imported</a> into an original array of column descriptions A, each column description within B is combined into the original array A by:
+              When an array of column descriptions B is <a>merged</a> into an original array of column descriptions A, each column description within B is combined into the original array A by:
```

Comment:
The thing I care about here is when and how we raise the incredibly useful validation error that flags up when someone's reordered their columns or inserted a new one, and the titles don't match. Can we have an example that shows how that works? (When I've been thinking about it, I've been thinking we need two modes for the merge, a 'strict' that throws validation errors and a 'weak' in which the metadata just gets merged together.)